tests/unit/lms/models/_mixins/public_id_test.py

```diff
@@ -56,6 +57,13 @@ def test_public_id_comparator_raises_for_bad_ids(self, db_session, bad_public_id
                 ModelTestHost.public_id == bad_public_id
             ).one_or_none()
 
+    @pytest.fixture(autouse=True, scope="class")
+    def create_model_test_host_table(self, db_engine):
+        ModelTestHost.__table__.drop(db_engine, checkfirst=True)
+        ModelTestHost.__table__.create(db_engine)
+        yield
+        ModelTestHost.__table__.drop(db_engine)
```

Comment:
On `main` `sqlalchemcy.orm.create_all()` is called by a test fixture, and I think all of the test modules must be imported (by `pytest`) before the fixture is executed, so `ModelTestHost` is registered with the SQLAlchemy `Base` class before `create_all()` is called so the `model_test_host` table gets created.

On this branch `create_all()` is called by `init_db.py` when `tox.ini` run `init_db.py` before running `pytest`. `init_db.py` imports `lms.models` but it doesn't import all the tests. So `ModelTestHost` isn't registered when `create_all()` gets called and the `model_test_host` table doesn't get created.

I've worked around this by adding this text fixture to create and drop this one table. This is why the tests in this class all need to be sent to the same test runner: if they were sent to different parallel runners then multiple runners would try to create this table.

I'm calling `drop()` after the `yield` because I want to delete the table after the test run, I don't want to leave the table behind in the test DB. But I thought I better also call `drop(checkfirst=True)` (which drops the table only if it exists) before calling `create()` as well just in case the table somehow didn't get dropped by a previous run (e.g. due to a crash or something), otherwise the tests could get broken.